docs(issue labeling): explain impact label (#17237)
renovatebot/renovate: docs/development/issue-labeling.md

Use [this search](https://github.com/renovatebot/renovate/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+-label%3Apriority-1-critical+-label%3Apriority-2-high+-label%3Apriority-3-medium+-label%3Apriority-4-low++-label%3Apriority-5-triage) to find any issues which are missing a priority label.

### Impact

<details>
    <summary>Impact</summary>

    impact:small
    impact:medium
    impact:large

</details>

Use these to assign a impact level to an issue.

> Impact means risk to the end users or their use cases.

Reviewer: ```suggestion
Impact means risk to the end users or their use cases.
It's used to identify which changes can be made relatively quickly versus those which require great care before merging, due to their chance of negatively impacting a wide number of users if there's a bug.
```

Use [this search](https://github.com/renovatebot/renovate/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+-label%3Apriority-1-critical+-label%3Apriority-2-high+-label%3Apriority-3-medium+-label%3Apriority-4-low++-label%3Apriority-5-triage) to find any issues which are missing a priority label.

### Impact

<details>
    <summary>Impact</summary>

    impact:small
    impact:medium
    impact:large

</details>

Use these to assign a impact level to an issue.
Do we mean?

- Impact on usability for end users
- Impact on maintainers, like "amount of hassle/work this takes to complete"

> - Impact as risk of ...

Author: Need help from the maintainers to explain what the labels are for.

The label descriptions on GitHub are:

- `impact:small` Change has a small impact
- `impact:medium` Change has a moderate impact on users/use cases
- `impact:large` Change has potentially large impact on users/use cases

Reviewer: At the moment impact == risk.

It's not meant to mean "amount of work"

Author: ```suggestion
Impact means risk to the end users or their use cases.
It does _not_ mean "amount of work for the maintainers".
```

Is this correct? Feel free to edit the suggestion so it matches what you mean. 😉
